e2e_test/source_inline/kafka/alter/alter_connector_props.slt

```diff
@@ -84,6 +109,25 @@ select count(*)  from rw_sources where definition ILIKE '%secret s2%' and name =
 ----
 1
 
+
+query I
+select count(*)  from rw_sources where definition ILIKE '%secret s2%' and name = 'plain_students_source';
+----
+1
```

Comment:
@zwang28 you can check the test that the non-shared source, not running, i.e. no fragment, is also updated.